### DISCUSSION TITLE
Is there a git glyph state between untracked and staged

### DISCUSSION THREAD

**Discussion by xzhang8102** [nvim-tree/nvim-tree.lua#1763] [Q&A] [answered]
This problem bothered me a lot. I'm just wondering is there a git glyph state between staged and untracked (i.e., in one file, some of the changes are staged and some of them are not. this situation can be done using `Gitsigns stage_hunk`).

**Comment by alex-courtis** [comment, marked as answer]
It looks like the status is `MM` e.g.

```sh
:; git status --porcelain=v1
 M .gitignore
M  cfg.yaml
MM config.mk
```

Both the staged and unstaged icons will be shown:
![20221122_092236 372x220](https://user-images.githubusercontent.com/1505378/203170029-8800725c-b625-4190-b6d7-5649b4a71566.png)

However... if you use the sign column for git status only the first status (staged) is shown:
![20221122_092327 338x182](https://user-images.githubusercontent.com/1505378/203170137-ac0bc0cc-2dfe-4c59-ac3d-45071679f6b5.png)

**Reply by xzhang8102** [reply]
Oh, I see. Thx for your answer!
